static/templates/settings/settings_checkbox.hbs

```diff
@@ -7,6 +7,11 @@
     </label>
     <label for="{{prefix}}{{setting_name}}" class="inline {{setting_name}}_label" id="{{prefix}}{{setting_name}}_label">
         {{label}}
+        {{#if label_parens_text}}
+        {{#tr}}
+        (<i>{label_parens_text}</i>)
```

Comment:
The text in parens is now translated separately from rest of the label, but I think it should be fine since both are different sentences.

Reply:
Yep that sounds correct!